tests/test_validate.py

```diff
@@ -0,0 +1,22 @@
+"""Test for sorting MappingSetDataFrame columns."""
+
+import unittest
+
+from sssom.constants import SCHEMA_DICT
+from sssom.parsers import parse_sssom_table, to_mapping_set_document
+from sssom.util import sort_df_rows_columns
+from sssom.validators import json_schema_validate
+from tests.constants import data_dir
+
+
+class TestValidate(unittest.TestCase):
+    """A test case for sorting msdf columns."""
+
+    def setUp(self) -> None:
+        """Test up the test cases with the third basic example."""
+        self.msdf = parse_sssom_table(f"{data_dir}/basic.tsv")
+
+    def test_validate(self):
```

Comment:
Lets try and add 4 good tests here.. I think this command may be the most important of all in the end :)